### DISCUSSION TITLE
Timing for Nornir Release (3.1.?) with ruamel.yaml 0.17

### DISCUSSION THREAD

**Discussion by peterthayer** [nornir-automation/nornir#739] [Q&A] [answered]
Hi All. 

We have a security issue with Python-ruamel-yaml-0.16.10 that is fixed in Python-ruamel-yaml-0.17.  However, the v0.17 of ruamel.yaml introduces non-backward compatible change to the object interfaces. To remove the requirement of using v0.16 we would have to make internal modifications to our nornir package, but we see this fix has already been made on the current nornir dev branch.

Do you have a timing for the next Nornir release containing this fix?  The team is a bit blocked.  We either need to roll our own or wait for the mainline release. 

Thanks for your help.

**Comment by ktbyers** [comment]
Any reason you can't just use the Nornir `develop` branch in the interim?

**Comment by dbarrosop** [comment]
I agree a release is due at this point. I will try to take the time this week to do so.

**Comment by dbarrosop** [comment, marked as answer]
I just released nornir 3.2.0

https://github.com/nornir-automation/nornir/releases/tag/v3.2.0
